common4j/src/main/com/microsoft/identity/common/java/BuildValues.java

```diff
@@ -0,0 +1,55 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.common.java;
+
+import androidx.annotation.NonNull;
+
+//Used as a wrapper for setting and accessing values through either the generated BuildConfig.java class,
+//or from parameters set via the NativeAuthPublicClientApplicationConfiguration.kt file
+public class BuildValues {
```

Comment:
I am looking to see if this file can be removed.

Reply:
I have looked at the file and will discuss how we can use that for our purposes. I will do that as part of separate ticket to unblock this merge.

Reply:
I suggest you address it in this PR if possible so that duplicated code never makes it into branch in the first place. It is always easy to forget about addressing it later.

Reply:
I will definitely do it next week in a separate PR. I just want this PR to be merged so we can create the pull requests for SignIn, SignUp and SSPR as they are dependent on this base PR.